packages/application-extension/src/index.ts

```diff
@@ -91,6 +115,13 @@ namespace CommandIDs {
   export const resolveTree = 'application:resolve-tree';
 }
 
+/**
+ * Are the left and right panels available on the current page?
+ */
+const sidePanelsEnabled: () => boolean = () => {
+  return PageConfig.getOption('notebookPage') === 'notebooks';
+};
+
```

Comment:
Currently, side panels are only available on Notebook page, but they will be available on other pages.
So perhaps this function should be renamed `isNotebookPage`, and the function `sidePanelsEnabled` will return `isNotebookPage`, at first.
This way we can include `NotebookTools` only for the notebook page, for example.

Reply:
solved with https://github.com/jupyter/notebook/pull/6487#pullrequestreview-1081113649